periodic_builds/notify_results.sh

```diff
@@ -0,0 +1,20 @@
+success_dir="/workspace/success/*"
+failed_dir="/workspace/failed/*"
+
+for f in $failed_dir
+do
+	printf "Sending email for failed file; $(basename $f)\n"
+	go run main.go \
+	  --subject="[Periodic Build Notification] $(basename $f)" \
+	  --receiver="snny@google.com" \
```

Comment:
nit on your comment: the email used in  https://github.com/datacommonsorg/mixer/blob/master/build/ci/cloudbuild.goldendiff.yaml is `datacommons-alerts@google.com` (`datacommons` instead of `datcom`). The version from that cloudbuild is what I just pushed.